### PR TITLE
Changed success and cancel url in stripe

### PR DESCRIPTION
**Pull request by codeyumm**

### CODE REVIEW

File: app/api/(billing)/checkout/route.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+  const { plan, userId } = await req.json();
 
   let priceId: string | undefined;
   let mode: "payment" | "subscription";
 
-  // Determine priceId and mode based on plan type
+  // priceId and mode based on plan type
   switch (plan) {
     case "lifetime":
       priceId = process.env.STRIPE_PRICE_ID_LIFETIME;
```
Comment (shreynpatel23) [inline app/api/(billing)/checkout/route.ts:15]: @codeyumm can we create constants for the mode and store then in a separate file. that is a good practice to follow. By this if we want to change it in future we know exactly where to change

File: app/api/(billing)/checkout/route.ts [hunk 2]
```diff
@@ -37,6 +36,13 @@ export async function POST(req: NextRequest) {
   }
 
   try {
+    const metadata = {
+      userId: String(userId),  // Ensure it's a string
+      planType: String(plan),
+    };
+
+    console.log('Metadata before sending to Stripe:', metadata);
```
Comment (shreynpatel23) [inline app/api/(billing)/checkout/route.ts:44]: Please remove console.log

File: app/api/webhook/route.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+const webhookSecret = process.env.STRIPE_WEBHOOK_SECRET!;
+
+export const POST = async (req: NextRequest) => {
+  if (req.method === 'POST') {
```
Comment (shreynpatel23) [inline app/api/webhook/route.ts:18]: no need of this If condition as the function name is POST so next always consider this as a post request.

**Review by shreynpatel23 — CHANGES_REQUESTED** [review]
@codeyumm there are a lot of console.log in your code if we can remove them, that would be good, Also I have suggested some changes please take a look at that.

**Comment by shreynpatel23** [comment]
@codeyumm I think you bramch is not rebased with main branch please do that as well

File: app/api/(billing)/checkout/route.ts (hunk 1 above)
Comment (codeyumm) [reply app/api/(billing)/checkout/route.ts:15]: Done

File: app/api/(billing)/checkout/route.ts (hunk 2 above)
Comment (codeyumm) [reply app/api/(billing)/checkout/route.ts:44]: done

**Review by shreynpatel23 — APPROVED** [review]

**Pull request merged**